uimaj-core/pom.xml

```diff
@@ -260,4 +260,36 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <!-- ************ Java 9 enablement ************** -->
+    <profile>
+      <id>java9</id>
+      <activation>
+        <jdk>[9,)</jdk>
+      </activation>
+      <dependencies>
+        <!-- 
+          - To avoid having to add the java.xml.bind module in Java 9 and higher, we simply
+          - directly add the correpsonding dependencies. This seems to be the way to go
+          - anyway since java.xml.bind seems to be bound to be removed from future JDKs.
+          - See: https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j/43574427
+          -->
+        <dependency>
+          <groupId>javax.xml.bind</groupId>
+          <artifactId>jaxb-api</artifactId>
+          <version>2.3.0</version>
```

Comment:
Do you think that depending on some earlier version might get rid of that problem with extra new lines creeping in?

Reply:
I have no idea if the newline issue is related to JAXB. I rather expect that changes in the XSLT code introduce the newlines because turning off the pretty-print option in one test case fixed it... but well, it might be worth a try.